6tisch-roll-enrollment-priority.mkd

```diff
@@ -1,7 +1,7 @@
 ---
 title: Controlling Secure Network Enrollment in RPL networks
 abbrev: join-metric
-docname: draft-ietf-roll-enrollment-priority-07
+docname: draft-ietf-roll-enrollment-priority-06
```

Comment:
Why is the new draft version lower than previous?

Reply:
I think that I goofed incrementing the number too many times.  -05 is what is posted.